contentcuration/contentcuration/frontend/RecommendedResourceCard/components/RecommendedResourceCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+
+  <KCard
+    title="sample title"
+    :headingLevel="2"
+    :to="sampleRoute"
+    layout="horizontal"
+    thumbnailScaleType="cover"
+    thumbnailDisplay="small"
+    :titleLines="3"
+    thumbnailSrc="https://domf5oio6qrcr.cloudfront.net/medialibrary/11525/0a5ae820-7051-4495-bcca-61bf02897472.jpg"
+  >
+    <template #aboveTitle>
+      <div>
+        <KIconButton
+          icon="practiceSolid"
+        />
+        <span>Practice</span>
+      </div>
+    </template>
+    <template #title>
+    </template>
+    <template #belowTitle>
+      <div class="bellow-title-style">
+        <p>
+          below title slot section for the KCard component:
+          below title slot section for the KCard component
+        </p>
+      </div>
+    </template>
+    <template #footer>
+      <div class="align-right-style">
+        <KIcon icon="openNewTab" class="card-icon-size" />
+        <KIcon icon="thumbDown" class="card-icon-size" />
```

Comment:
These should be `KIconButton`s. As they are meant to elicit an action. Please note that the action on click is out of scope for the task assigned.